docs/index.md

```diff
@@ -10,10 +10,27 @@ Welcome to the Zigbee2mqtt documentation! Besides the documentation, you can als
 
 ### Getting started
 * [What do I need?](getting_started/what_do_i_need.md)
-* [Flashing the CC2531 USB stick](getting_started/flashing_the_cc2531.md)

```

Comment:
I think we should keep the flashing and pairing devices page in the getting started, as this section should follow what steps to execute on how to get zigbee2mqtt up and running. (https://www.zigbee2mqtt.io/getting_started/what_do_i_need.html#whats-next).

Also regarding how-tos I think it's nice to keep them together.

Is your problem solved by just moving the `Running zigbee2mqtt on..` below `Getting started`?

Reply:
Sure! Also having the bare metal linux option bullet pointed so it's not overlooked :)